dashboard/admin.py

```diff
@@ -163,10 +163,16 @@ class PersonAdoptPracticeAdmin(admin.ModelAdmin):
         models.CharField: {'widget': forms.CheckboxSelectMultiple(choices=NONNEGOTIABLE_OPTION)},
         models.TextField: {'widget': Textarea(attrs={'rows':4, 'cols':40})},
     }
-    list_display = ('id', 'date_of_adoption', '__unicode__', 'verification_status', 'non_negotiable_check', 'verified_by')
+    list_display = ('id', 'date_of_adoption', 'date_of_verification', '__unicode__', 'verification_status', 'non_negotiable_check', 'verified_by')
```

Comment:
date of adoption should be removed and instead replaced with date of verification.